.github/workflows/checks.yml

```diff
@@ -24,12 +24,14 @@ jobs:
       with:
         python-version: 3.8
 
-    # TODO(c-bata): Remove the version constraint on SQLAlchemy
     - name: Install
       run: |
         python -m pip install -U pip
         pip install --progress-bar off -U .[checking]
+        # TODO(c-bata): Remove the version constraint on SQLAlchemy
         pip install "sqlalchemy<2.0.0"
+        # TODO(not522): Remove the version constraint on NumPy
+        pip install "numpy<1.24"
```

Comment:
Is this necessary to resolve mypy error? It seems `Checks (Integration)` passes without version constraint (https://github.com/Alnusjaponica/optuna/pull/6).

Reply:
Oh, I forget that `--warn-unused-ignores` is not applied in checks CI. Thank you for your careful review.